.github/workflows/macos-release-candidate.yml

```
@@ -89,7 +110,7 @@ jobs:
     - name: Upload release artifacts to Google Cloud Storage bucket
       run: |
         gsutil rm -rf ${{ env.BUCKET_PATH }} || true
-        gsutil cp -r dist/installers ${{ env.BUCKET_PATH }}
+        gsutil cp dist/installers/*.dmg* dist/installers/latest-mac.yml ${{ env.BUCKET_PATH }}
```

```suggestion
        gsutil cp dist/installers/Brim-*.dmg* dist/installers/Brim-*-mac.zip dist/installers/latest-mac.yml ${{ env.BUCKET_PATH }}
```

This this is a change I needed to make in my fork to get auto-updates working on macOS. I'd previously not realized the significance of the `.zip` artifact and it only became clear when auto-update failed and I saw the message in the log indicating it was pining for that file.

While I'm at it, I made the glob for the DMG-centric artifacts a little more strict, just for kicks.